src/validation.h

```diff
@@ -35,6 +35,7 @@
 #include <stdint.h>
 #include <string>
 #include <thread>
+#include <unordered_map>
```

Comment:
b44fea2c it looks like these two includes should be added to `node/blockstorage.{h,cpp}` -- or only the header file if you inline `UpdatePruneBlocker()` in the h file -- instead of the `validation.{h, cpp} files`

Reply:
Done